index.html

```diff
@@ -1220,11 +1448,8 @@ <h2 class="ui header">Glyphs <em class="fontname" style="display: none;">FontNam
     }
     bitmapsOutput += '};\n\n'
 
-    // Glyphs
-    // The last glyph has a ',' too much at the end, we need to remove it (it's easier than to avoid putting it in the first place)
-    glyphs[glyphs.length - 1] = glyphs[glyphs.length - 1].replace('},', '} ')
     let glyphsOutput = 'const GFXglyph ' + name + 'Glyphs[] PROGMEM = {\n'
-    glyphsOutput += glyphs.join('\n') + '\n};\n\n'
+    glyphsOutput += '{' + glyphs.join("\n{") + '\n};\n\n'
```

Comment:
`glyphs` is an array of arrays now so I don't think that works 🤔 ?

Reply:
Cannot tell looking to code, but if it would not...
Anyway I changed your logic here a bit - I am using array with data as I am changing offsets too so maybe it looks a bit different then before, also formatting was dropped at least for xored (simplicity).

Reply:
You are right, but with plain JS you can do miracles :-)
glyphs
```
(93) [Array(6), Array(6), Array(6), ...]
0: (6) [45, 4, 34, 10, 5, "-33 },// 0x21 '!'"]
1: (6) [-102, 13, 12, 16, 2, `-33 },// 0x22 '"'`]
```
glyphs.join("\n{")
`'45,4,34,10,5,-33 },// 0x21 \'!\'\n{-102,13,12,16,2,-33 },// 0x22 \'"\'\n{33,25,34,26,0,-33 },// 0x23 \'#\'\n{193,22,40,25,2,-35 },// 0x24 \'$\'\n{439,36,34,40,3,-33 },// 0x25 \'%\'\n{435,28,34,31,2,-33 },// 0x26 \'&\'\n{227,4,12,7,2,-33 },// 0x27 \'\'\'\n{268,11,43,15,3,-33 },// 0x28...`

Would tell there is default toString() for small arrays and comma default separator(?) - by fortune there is nothing like `[object]\n[object]...` one would expect 👍
Also `glyphs.toString()` does almost same except LFs.

And if someone want to play with, may check [ArduinoSimulator repo](https://github.com/eltomjan/ETEhomeTools/tree/master/ArduinoSimulator) I used to build / test Arduino version - simple port of Arduino to C# (form) / C++ made in good old VS 2008 Express (?)
Or pack / improve compression in C# version [GFXpack repo](https://github.com/eltomjan/ETEhomeTools/tree/master/CSharp/GFXpack), great is CharBox/Row there show nice text version of data inside, you can edit them its byte array or say Xor even in Immediate window.